docs/UserGuide.md

```diff
@@ -230,6 +232,7 @@ Updates the contact information of a participant in the app.
 * `<phone number>` should be **numeric**, **non-empty** and **not longer than 15 digits**.
 * `<email>` should be **alphanumeric**, **non-empty** and **not longer than 64 characters**.
 * `<address>` should be **non-empty** and **not longer than 64 characters**.
+* `<tag>` should be **not longer than 64 characters**.
```

Comment:
Are we changing this part?

Reply:
i think we need to delete off under all cautions cause there is no code to actually enfore this rule